### PR TITLE
Add parent email confirmation when added to a student account

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Preview can be seen at http://localhost-studio.code.org:3000/rails/mailers/parent_mailer/parent_email_added_to_student_account_preview

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dju90** [comment]
Email body looks good, did a double read-through to check for typos

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -250,6 +250,7 @@ en:
       %{name}
   parent_mailer:
     student_associated_subject: 'Login information for Code.org'
+    parent_email_added_subject: 'Welcome to Code.org'
```
Comment (dju90) [inline dashboard/config/locales/en.yml]: Can you add an exclamation point to the end of the subject? 'Welcome to Code.org!' (same as the teacher email)

Comment (bethanyaconnor) [reply dashboard/config/locales/en.yml]: Done!

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+%p
+  You and your student can also participate in Code Break each Wednesday at 10 am PST / 1 pm ET.
+  Code Break is a live, interactive classroom with weekly challenges to engage students of all abilities,
+  even those without computers. Learn more about
+  %a{href:"https://code.org/break"}Code Break.
+
+%p
+  Don’t have a computer? We can suggest some options that work great on mobile phones or tablets too!
+  %a{href:"https://code.org/athom#apps"}See smartphone options.
```
Comment (Hamms) [inline dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml]:
```suggestion
  %a{href:"https://code.org/athome#apps"}See smartphone options.
```

File: dashboard/test/mailers/previews/traverse_mail_previews_test.rb [hunk 3]
```diff
@@ -4,9 +4,14 @@ class TraverseMailPreviewsTest < ActiveSupport::TestCase
...
       require file
-      Object.const_get('Pd::' + file.scan(/\/pd_([\w_]+).rb/).first.first.camelize)
+      if file.scan(/\/pd_([\w_]+).rb/).empty?
+        Object.const_get(file.scan(/\/([\w_]+).rb/).first.first.camelize)
+      else
+        Object.const_get('Pd::' + file.scan(/\/pd_([\w_]+).rb/).first.first.camelize)
+      end
     end
 
+    puts classes
```
Comment (Hamms) [inline dashboard/test/mailers/previews/traverse_mail_previews_test.rb]: stray debug statement

**Review by Hamms — APPROVED** [review]
couple nits, otherwise LGTM!

File: dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml (hunk 2 above)
Comment (bethanyaconnor) [reply dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml]: Nice catch!

**Pull request merged**